test/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/MsBuildExecutor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using BenchmarkDotNet.Toolchains.Results;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class MsBuildExecutor : IExecutor
```

Comment:
@cesarbs Sure! I have got something up and running: [commit](https://github.com/dotnet/BenchmarkDotNet/commit/c7aba11656357ac052b4bd5ba31e36e496319415) but not final yet, see the [description](https://github.com/dotnet/BenchmarkDotNet/issues/300#issuecomment-275353434)

This week I am in Redmond for the MS LEAP, so we can even meet in person and talk about it  ;) btw were you able to run this auto-generated dll? When I tried similar approach (to execute with process.Start) I got some missing dlls exceptions (the directory no longer contains the executable). That's why I have switched to `dotnet run` executor (but it has some limitaitons)